src/components/Layout/Layout.js

```diff
@@ -1,27 +1,38 @@
-import React from "react";
+import React, { useState } from "react";
 import classes from "./Layout.module.css";
 import Side from "../Side/Side";
 import useFarmlands from "../../hooks/useFarmlands";
+import useCompanies from "../../hooks/useCompanies";
 import FarmlandsList from "../FarmlandsList/FarmlandsList";
+import CompaniesList from "../CompaniesList/CompaniesList";
 
 const Layout = () => {
-  const { farmlands } = useFarmlands();
-  const handlerSelectSide = (target) => {
-    console.log(target);
-  };
+	const { farmlands } = useFarmlands();
+	const { companies } = useCompanies();
+	const [selectedList, setSelectedList] = useState("farmlands");
+	const handlerSelectSide = (target) => {
+		setSelectedList(() => {
+			return target;
```

Comment:
Maybe we can just use the simple update state form: `setSelectedList(target)`, because here the next state doesn't depend on the previous one. What do you think?